requirements.txt

```diff
@@ -0,0 +1,5 @@
+git+https://github.com/canonical/operator/#egg=ops
```

Comment:
You may want to try using `ops` directly

Reply:
Just a habit I guess.
I did and it worked.